### ISSUE TITLE
Incorrect license detection for Python package build with poetry package manager

### ISSUE THREAD

**Issue by pilosus** [pilosus/pip-license-checker#81]
[aiofiles](https://pypi.org/project/aiofiles/) license on PyPI is detected as: 

```
Other/Proprietary License (Apache 2.0)
```

although the `pyproject.toml` clearly specifies [Apache 2.0](https://github.com/Tinche/aiofiles/blob/v0.7.0/pyproject.toml#L6), just like the [LICENSE](https://github.com/Tinche/aiofiles/blob/v0.7.0/LICENSE) file.

It could happen if `poetry build` somehow inserts the wrong trove classifier `License :: Other/Proprietary License` into project's metadata.

**Comment by pilosus** [comment]
> It could happen if poetry build somehow inserts the wrong trove classifier License :: Other/Proprietary License into project's metadata.

This is exactly what happens:

```
➜  aiofiles-0.7.0.dist-info git:(master) pwd
/home/vitaly/.pyenv/versions/3.9.6/envs/py396/lib/python3.9/site-packages/aiofiles-0.7.0.dist-info

➜  aiofiles-0.7.0.dist-info git:(master) grep -r 'License' METADATA
License: Apache 2.0
Classifier: License :: Other/Proprietary License
```

The `poetry` docs [say](https://python-poetry.org/docs/pyproject/#classifiers):

> The license property will also set the License classifier automatically.

To make that automation happen properly, the [license](https://python-poetry.org/docs/pyproject/#license) property has to contain an SPDX Open Source License Registry identifier (see the list in poetry's [source code](https://github.com/python-poetry/poetry-core/blob/8d5e5c5d070940736ab72c5dec09cd7deab07cf3/poetry/core/spdx/license.py#L87)).

The `aiofiles` uses incorrect license identifier:

```diff
- license = "Apache 2.0"
+ license = "Apache-2.0"
```

A pull request to `aiofiles` to fix that is needed :-)

**Comment by pilosus** [comment]
Already [fixed](https://github.com/Tinche/aiofiles/issues/106) in pre-release version `0.8.0dev0`.
Nothing to fix from our end.
